library/Preprocessor.php

```diff
@@ -42,10 +42,11 @@ class Webgrind_Preprocessor
      * @param string $outFile File to write preprocessed data to
      * @return void
      */
-    static function parse($inFile, $outFile)
+    static function parse($inFile, $outFile, $readMemory = false)
     {
+        $costPattern = $readMemory ? "%*d %d" : "%d";
```

Comment:
Could you explain the reason for this pattern?
The `%*d` is skipping the number correct?
What number is in the second / third positions?

Reply:
Yes, every "cost line" looks like this:
`4 936 160` == `{line number} {time cost in usec} {memory cost in bytes}`

When we want to read memory, we effectively need to skip the time cost and read the last number (which is normally unread).